### ISSUE TITLE
Setting Flags using Mailkit on a Ipswitch IMail server

### ISSUE THREAD

**Issue by kkay2112** [jstedfast/MailKit#140]
Have you seen any issues when setting flags on mail messages using the Ipswitch Imail server?
I'm seeing the command submitted to the server, but no Flags are being passed.  And when catching the MessageFlagsChanged event there are No flags set.

Is there anyway to submit a low level command to the server through Mail Kit?  

---

```
                               inbox.RemoveFlags(i, MessageFlags.Seen, false);

                                inbox.MoveTo(i, archiveFolder);

                                inbox.SetFlags(i,MessageFlags.Deleted,false);
```

---

C: A00000011 STORE 1 -FLAGS ()
S: \* 1 FETCH (FLAGS ())
S: A00000011 OK STORE Completed
S: \* 3 EXISTS
S: \* OK [UNSEEN 2] Message 1 is the first unseen message
C: A00000012 COPY 1 Archive
S: A00000012 OK Copy completed
C: A00000013 STORE 1 FLAGS ()
S: \* 1 FETCH (FLAGS ())
S: A00000013 OK STORE Completed

**Comment by jstedfast** [comment]
I just released MailKit 1.0.5 to NuGet this morning with this fix.

**Comment by kkay2112** [comment]
Great news, Thanks for the update.

Regards,

Kris Kay
Vice President
Enterprise Development
Integration Point
Phone: (704) 576-3678 1123
Find Us:  Web<http://www.integrationpoint.com/>  Blog<http://www.integrationpoint.com/globaltradenews>  Twitter<http://www.twitter.com/GlobalTradeNews>  LinkedIn<http://www.linkedin.com/groupRegistration?gid=2045761>

[cid:image001.jpg@01D02B2E.5EBC40B0]<http://www.integrationpoint.com/>
